packages/labs/task/package.json

```diff
@@ -57,12 +56,14 @@
         "build:ts:types",
         "build:rollup",
         "../../lit:build",
-        "../../reactive-element:build"
+        "../../reactive-element:build",
```

Comment:
do we need lit and reactive-element anymore?

Reply:
For tests, yes